backend/bin/deal-observer-backend.js

```diff
@@ -57,10 +55,10 @@ const observeActorEventsLoop = async (makeRpcRequest, pgPool) => {
       Sentry.captureException(e)
     }
     const dt = Date.now() - start
-    console.log(`Loop "${LOOP_NAME}" took ${dt}ms`)
+    console.log(`Loop "Observe actor events" took ${dt}ms`)
```

Comment:
I propose the following log for consistency with the telemetry point name.

```suggestion
    console.log(`Loop "Observe built-in actor events" took ${dt}ms`)
```